scripts/install_rkt.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -ex
+
+RKT_VERSION="v1.0.0"
+DEST_DIR="/usr/local/bin"
+
+wget https://github.com/coreos/rkt/releases/download/$RKT_VERSION/rkt-$RKT_VERSION.tar.gz
+tar xzvf rkt-$RKT_VERSION.tar.gz
+sudo cp -R rkt-$RKT_VERSION $DEST_DIR
```

Comment:
That won't work, we will need to copy all the stage1 ACIs too. I think I should enumerate everything I want to copy.